gdal/ogr/ogrsf_frmts/pgeo/ogr_pgeo.h

```diff
@@ -61,6 +61,8 @@ class OGRPGeoLayer CPL_NON_FINAL: public OGRLayer
 
     int                *panFieldOrdinals;
 
+    int                 m_bEOF;
```

Comment:
We can modernize a bit:
```suggestion
    bool                m_bEOF = false;
```